exercises/prime-factors/prime_factors_test.exs

```diff
@@ -62,4 +62,16 @@ defmodule PrimeFactorsTest do
   test "93819012551" do
     assert PrimeFactors.factors_for(93819012551) == [11, 9539, 894119]
   end
+
+  @tag :optional
+  @tag timeout: 120000
+  test "performance optimizations" do
+    %{minute: original_minutes, second: original_seconds} = DateTime.utc_now
+    PrimeFactors.factors_for(10000000055) == [5, 2000000011]
```

Comment:
I don't think this needs to be 2 test cases, code written up to that point should have been tested enough that it will get the correct answer. Even then, the order of the assertions will give you what you want anyway. These are optional tests so there's no value I see from splitting them